src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -1,21 +1,444 @@
-import {createMockContext, makeCli} from './fixtures'
+import fs from 'fs'
+import process from 'process'
+import * as stream from 'stream'
+import util, {InspectOptions} from 'util'
+
+import axios from 'axios'
+import FormData from 'form-data'
+import JSZip from 'jszip'
+
+import {API_KEY_ENV_VAR, AWS_DEFAULT_REGION_ENV_VAR, CI_API_KEY_ENV_VAR} from '../constants'
+import {LambdaFlareCommand, writeFile, zipContents} from '../flare'
+import {requestAWSCredentials} from '../prompt'
+
+import {createMockContext, makeCli, mockAwsCredentials} from './fixtures'
+
+// Constants
+const MOCK_FOLDER_PATH = './mock_folder'
+const MOCK_FILE_PATH = 'function_config.json'
+const MOCK_ZIP_PATH = 'output.zip'
+const MOCK_API_KEY = 'test-api-key'
+const VALID_INPUT = ['lambda', 'flare', '-f', 'func', '-r', 'us-west-2', '-c', '123', '-e', 'test@test.com']
+const MOCK_CONFIG = {
+  Environment: {
+    Variables: {
+      DD_API_KEY: MOCK_API_KEY,
+      DD_SITE: 'datadoghq.com',
+      DD_LOG_LEVEL: 'debug',
+    },
+  },
+  FunctionArn: 'arn:aws:lambda:us-east-1:123456789012:function:some-function',
+  FunctionName: 'some-function',
+}
+
+// Mock util.inspect to remove color codes
+const originalInspect = util.inspect
+jest.spyOn(util, 'inspect').mockImplementation((object: any, options?: InspectOptions) => {
+  return originalInspect(object, {...options, colors: false})
+})
```

Comment:
I suggested mocking globally but this is doing it in this file, what is the advantage of doing it here? and, why are we spying on it instead of mocking it? Wouldn't it make more sense to mock it like the module `chalk` was mocked globally?

Reply:
Converted it to a global mock in `__mocks__/chalk.js`